tests/integration/test_models/test_full_battery_models/test_lithium_ion/test_ecm_split_OCV.py

```diff
@@ -0,0 +1,58 @@
+#
+# Test that the model works with an example parameter set
+#
+import pybamm
+import numpy as np
+
+
+class TestECMSplitOCVModel:
+    def test_run_model_with_parameters(self):
```

Comment:
This test does not asset anything. If a test does not assert anything then the only thing that gets tested is that the code does not crash and the coverage is artificially high.

A lot of the other models use something like this:
```
    def test_basic_processing(self):
        model = pybamm.equivalent_circuit.Thevenin()
        modeltest = tests.StandardModelTest(model)
        modeltest.test_all()
```
Is it possible to set that up instead?

Reply:
Done